articles/getting-started-using-admob-with-react-native-expo/index.md

```diff
@@ -0,0 +1,484 @@
+
+![admob dashboard](getting-started-using-admob-with-react-native-expo/1-admob-all-apps-dashboard.png)
+
+From the dashboard, we need to create a new app. For that, we need to go to [create](https://apps.admob.com/v2/apps/create) a new Admob app console and select the required options as directed in the screenshot below:
+
+![create new admob app choose platform](getting-started-using-admob-with-react-native-expo2-create-new-admob-app-choose-platform.png)
+
+After clicking on 'Continue', we need to fill the **App name** and subsequently click on the 'ADD APP' button as shown in the screenshot below:
+
+![naming admob app](getting-started-using-admob-with-react-native-expo/3-naming-admob-app.png)
+
+Next, we need to add the Ad Unit as directed in the screenshot below:
+
+![create add unit](/engineering-education/getting-started-using-admob-with-react-native-expo/4-create-add-unit.png)
+
+The Admob console generally provides 5 types of ads to be integrated. For this tutorial, we are going to focus on banner ads and rewarded ads. The ad types to be selected are shown in the screenshot below:
+
+![admob ad type](/engineering-education/getting-started-using-admob-with-react-native-expo/5-admob-ad-type.png)
+
+First, we are going to create a **Banner ad** as directed in the screenshot below:
+
+![create banner ad](/engineering-education/getting-started-using-admob-with-react-native-expo/6-create-banner-ad.png)
+
+Here, we have named our Ad unit name as 'header ads'. Now after clicking on 'CREATE AD UNIT', we will receive an **Ad Unit ID** as shown in the screenshot below:
+
+![success create ad unit](/engineering-education/getting-started-using-admob-with-react-native-expo/7-success-create-ad-unit.png)
+
+Now, we will be able to get the Ad Unit ID from dashboard as well:
+
+![Ad unit dashboard](/engineering-education/getting-started-using-admob-with-react-native-expo/8-ad-unit-dashboard.png)
+
+Now, we are going to repeat the same process and create a Reward Ad. For that, we can customize the **Ad Unit name** and **Amount** as directed in the screenshot below:
+
+![create reward ad](/engineering-education/getting-started-using-admob-with-react-native-expo/9-create-reward-ad.png)
+
+After clicking on 'CREATE AD UNIT', we will get the **Ad Unit ID** for the **Reward Ads** as well. Now, we will be able to see two Ad units in our console:
+
+![fininsh create ad unit](/engineering-education/getting-started-using-admob-with-react-native-expo/10-finish-create-ad-unit.png)
+
+### STEP 2: Setup Admob on Expo
+To set up Admob on Expo, we will need to export the previous project from the Expo snack. This is due to the fact that we are not authorized to access the `app.json` to use in environment variables. Therefore, we need to export the entire expo project as directed in the screenshot below:
+
+![export expo project from snack](/engineering-education/getting-started-using-admob-with-react-native-expo/11-export-project-from-snack.png)
```

Comment:
The topic of your previous article has nothing to do with Admobs. Since you are reusing the App you created from the previous article, I think you should provide the app's code as a starter code, and then refer to your previous article and say that if you'd like to learn how to build the starter code provided here, refer to my previous article.

Reply:
AKA, The completed version of your previous article's app.

Reply:
fixed